_posts/2020-08-19-pulp-ansible-0.2.0-is-generally-available.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Pulp Container 2.0.0 is Generally Available
+author: Melanie Corr
+tags:
+  - release
+---
+
+# Pulp Ansible 0.2.0 Release Announcement
+
+The Pulp team is pleased to announce the release of the Pulp Ansible 0.2.0 plugin.
+
+## About the Pulp Ansible content plugin
+
+If you’re an Ansible user and do not want to host your private content on Ansible Galaxy, you can add the Pulp Ansible plugin to Pulp. You can then mirror the public Ansible content that you require and use Pulp as an on-premise platform to manage and distribute a scalable blend of public and private Ansible roles and collections across your organization.
+
+With the Pulp Ansible plugin, you can complete the following actions via the Pulp API. Ansible content includes both roles and collections:
+
+* Mirror a subset of Ansible content on-premise
+* Mirror all of Ansible Galaxy’s content on-premise
+* Store private Ansible content on-premise
+* Install Ansible content from pulp_ansible using the ansible-galaxy CLI
+* Version Ansible content over time and rollback if necessary
+* Support for the new multi-role content type from Galaxy
+
+For more information, see the [roles](https://pulp-ansible.readthedocs.io/en/latest/workflows/roles.html) and [collections](https://pulp-ansible.readthedocs.io/en/latest/workflows/collections.html) workflows in the Pulp Ansible documentation.
+
+## Release Features
+
+In this release, which is compatible with Pulp 3.6, the following functionality has been added:
+
+* Users can now associate a remote with a repository and automatically use this remote when syncing the repository. [#7194](https://pulp.plan.io/issues/7194)
+
+As part of this release, artifacts are no longer used when importing collections [#6718](https://pulp.plan.io/issues/6718)
```

Comment:
Is this "temporary artifacts are no longer persisted"?
In the end, i sure hope that an artifact is used to represent the collection. ;)

Reply:
Is the list bullet missing here?